google/datalab/ml/_cloud_models.py

```diff
@@ -168,12 +168,13 @@ def get_version_details(self, version_name):
     name = ('%s/versions/%s' % (self._full_model_name, version_name))
     return self._api.projects().models().versions().get(name=name).execute()
 
-  def deploy(self, version_name, path):
+  def deploy(self, version_name, path, runtime_version='1.0'):
```

Comment:
Should the default be "not set"? I think if not set, CloudML will choose a version.